sdk/appconfiguration/app-configuration/package.json

```diff
@@ -55,7 +55,7 @@
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "lint:fix": "eslint -c ../../.eslintrc.old.json src test samples --ext .ts --fix",
-    "lint": "eslint -c ../../.eslintrc.old.json src test --ext .ts -f html -o template-lintReport.html || exit 0",
```

Comment:
Please do not update `package.json` files in this PR. We can update them linter when packages are verified to be free from linting errors and then we can call the `lint` script in the `build` one.

Reply:
I will make this change. I am closing this PR and will open another one with just the changes in removing ESlint

Reply:
Thank you Praveen! Great changes! We'll have this in for sure. The PR separation is appreciated 🙌